src/here-xyz.ts

```diff
@@ -215,7 +197,8 @@ async function execInternal(
     contentType: string,
     data: any,
     token: string,
-    gzip: boolean
+    gzip: boolean,
+    setAuthorization: boolean
```

Comment:
is this required?

Reply:
Yes, setAuthorization flag was added and authorization was made optional to access public URLs of viewer link in CLI.